extensions/interactions/NumericInput/protractor.js

```diff
@@ -32,6 +32,8 @@ var expectInteractionDetailsToMatch = function(elem) {
 var submitAnswer = function(elem, answer) {
   elem.element(by.tagName('oppia-interactive-numeric-input')).
     element(by.tagName('input')).sendKeys(answer + '\n');
```

Comment:
Yeah, the issue was the submitAnswer method wouldn't get called for TextInput, NumericInput and one more interaction I don't remember. These interactions had the ng-form attribute.